comunicatii/src/main/java/code4romania/czl/scrapers/comunicatii/ScraperComunicatii.java

```diff
@@ -26,216 +26,282 @@
-                                                break;}
-                                                case "a": {
-                                                    description.append(getTextFromNode(cNode));
-                                                    cBean.addDocument(cNode.attr("href"), getTextFromNode(cNode));
-                                                }
-                                            }
-
-
-                                        });
-                                    });
-                                    cBean.setDescription(description.toString());
-                                    cBean.setTitle(getTextFromNode(groupedElements.get(0)));
-
-                                    extractDate(cBean);
-                                    String feedback = extractFeedbackDate(cBean.getDescription(), DATE_FEEDBACK_REGEX);
-                                    String start = extractFeedbackDate(cBean.getDescription(), DATE_PUBLICARE_REGEX);
-
-                                    cBean.setFeedback_days(calculateFeedbackDays(start, feedback));
-                                    cBean.setType(deductType(description.toString()));
-//                                    System.out.println(cBean.toJson());
-
-                                    cBean.setIdentifier(new UID().toString()+":"+cBean.getType()+":"+cBean.getDate());
-                                    groupedElements.clear();
-                                }else{
-                                    if(StringUtils.isNotBlank(item.toString())) {
-                                        groupedElements.add(item);
-                                    }
-                                }
-
-                            });
-
-//                    System.out.println(item)
-                }
-        );
-
-        outputList.forEach(el -> {
-            System.out.println(el.toJson());
-            System.out.println("sending");
-                    try {
-                        String result = post("http://czl-api.code4.ro/api/publications/", el.toJson());
-                        System.out.println(result);
-                    } catch (IOException e) {
-                        e.printStackTrace();
-                    }
+	private static final String DATE_REGEX = "([0-2][0-9]|30|31)\\.[0|1][0-9]\\.[0-9]{4}";
+	private static final String DATE_PUBLICARE_REGEX = "(publicat).+"
+			+ DATE_REGEX;
+	private static final String DATE_FEEDBACK_REGEX = "(comentarii).+"
+			+ DATE_REGEX;
+
+	public static void main(String[] args) {
+
+		List<ComunicatiiBean> outputList = new ArrayList<ComunicatiiBean>();
+
+		String url = "http://www.comunicatii.gov.ro/?page_id=3517";
+		Document document = null;
+		try {
+			document = Jsoup.connect(url).get();
+			
+			// document =
+			// Jsoup.parse(ScraperComunicatii.class.getResourceAsStream("/Proiecte de Acte Normative _ Comunicatii.gov.ro.html"),
+			// "UTF-8","");
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
In cazul in care aceasta exceptie e aruncata, sa zicem ca scraperul nu s-a putut conecta la site, documentul o sa fie `null` iar scraperul va incerca sa continue procesarea. Logic ar fi ca procesarea sa se intample doar in cazul in care documentul HTML a fost descarcat cu succes.